stubs/app/Actions/Jetstream/DeleteUserWithTeams.php

```diff
@@ -56,4 +58,17 @@ protected function deleteTeams($user)
             $this->deletesTeams->delete($team);
         });
     }
+
+    /**
+     * Delete the connected accounts and account associations attached to the user.
+     *
+     * @param  mixed  $user
+     * @return void
+     */
+    protected function deleteConnectedAccounts($user)
+    {
+        if (method_exists($user, 'connectedAccounts')) {
```

Comment:
@taylorotwell whats best practice for Laravel here? `method_exists` doesn't care about the methods visibility. Would you prefer `is_callable`?